src/python/pants/backend/helm/target_types.py

```diff
@@ -123,6 +130,18 @@ class HelmChartLintStrictField(TriBoolField):
     help = "If set to true, enables strict linting of this Helm chart."
 
 
+class HelmChartRepositoryField(StringField):
+    alias = "repository"
+    help = softwrap(
+        """
+        Repository to use in the Helm registry where this chart is going to be published.
+
+        If no value is given and `default-registry-repository` is undefined too, then the chart
```

Comment:
Suggest including the options scope here, for clarity:
> If no value is given and `default-registry-repository` is undefined too,

`... [helm].default-registry-repository ...`